english/pinch/magic/harry-potter.md

```diff
@@ -5,3 +5,4 @@ You try to remember some magic from the harry potter books
 that you have read so many times by now.
 
 [try apparating](../../movie-ripoffs/hogwarts/hogwarts.md)
+[try to escape by burning down the marshmallow walls](../../movie-ripoffs/hogwarts/burn-walls.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.